website/src/components/product/CustomerLogosSectionAnimated.tsx

```diff
@@ -177,9 +177,10 @@ interface Props {
     showButton: boolean
     className: String
     showSection: boolean
+    noCta?: boolean
```

Comment:
This seemed like the least disruptive way of getting what I needed (no extra section and no CTA buttons anywhere on desktop or mobile), but open to suggestions.